processor/processor.go

```diff
@@ -1656,6 +1665,10 @@ func (proc *Handle) processJobsForDest(partition string, subJobs subJob) *transf
 			continue
 		}
 
+		if source.SourceDefinition.Category != "" && !strings.EqualFold(source.SourceDefinition.Category, sourceCategoryWebhook) {
```

Comment:
`webhook` cateogory sources are event stream sources. We shouldn't be filtering `webhook`.
 rETL and ETL has `warehouse` and `singer-protocol`

Reply:
yeah we are only selecting sources where category is empty string or `webhook` rest all we are ignoring

Reply:
my bad. read the code wrong